src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -110,6 +110,7 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
+        System.out.println(logic.getFilteredPersonList());
```

Comment:
Is this line for debugging? If so, it should be removed

Reply:
oh yeah thanks mate I didn't catch that